src/components/SchemaView/SchemaViewMain.vue

```diff
@@ -242,18 +242,21 @@ export default {
       return relTable.group ? relTable.group : relTableName;
     },
     getLayoutConfig(edges) {
-      const nodeSpacing = edges.length * 5;
       const config = {
-        type: 'comboForce',
+        type: 'force',
```

Comment:
This should not be changed to `force`. The schema view uses `combo` to represent RDFGraph. It can cause issue and incorrect layouting when RDF presents.